layouts/default.vue

```diff
@@ -5,6 +5,11 @@
 </template>
 
 <style>
+@import url('https://fonts.googleapis.com/css2?family=Nunito&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Saira+Semi+Condensed:wght@600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Saira+Semi+Condensed:wght@800&display=swap');
```

Comment:
You have 2 fonts which are identical, only the `wght` differs which I believe implies for `font-weight`.
So, would it make any difference if you import any one and configure the `font-weight` in the CSS file itself?

Just curious

Reply:
@SBagaria2710 Yes, you are right. We have same `font` with different `font-weight`. But the issue is, it is not working just by importing that particular `font` & giving different `font-weights`. If possible to you can you please look into it. May be I am missing something.

Reply:
That shouldn't be the case. I'll check it and will update you.
For now, approving it from my end.

Reply:
> 
> 
> That shouldn't be the case. I'll check it and will update you.
> For now, approving it from my end.

OK thank you